app/bundles/EmailBundle/Entity/Email.php

```diff
@@ -1063,6 +1063,11 @@ private function decodeAmpersands(&$content)
      */
     public function getReadPercentage($includevariants = false)
     {
-        return ($includevariants) ? round($this->getSentCount(true) > 0 ? ($this->getReadCount(true)/($this->getSentCount(true))*100) : 0, 2) : round($this->getSentCount() > 0 ? ($this->getReadCount()/($this->getSentCount())*100) : 0, 2);
+        if ($this->getSentCount($includevariants)) {
+            return $this->getReadCount($includevariants)/($this->getSentCount($includevariants))*100;
+        }
+        else {
```

Comment:
It looks much better. This is very minor thing, but to keep the code style the same in the whole Mautic code base, could you change that to `} else {` (in one line)?

There is also pre-commit hook which can keep the code style in line automatically for you: https://github.com/mautic/mautic/blob/staging/.github/CONTRIBUTING.md#coding-standards